src/components/Alerts/index.module.scss

```diff
@@ -1,42 +1,79 @@
 @import 'sizes.scss';
 @import 'colors.scss';
 
-.alert {
-    color: $brand-color-darkest;
-    &:not(.important) {
-        background: desaturate(lighten($brand-color, 30%), 60%);
-        .container {
-            background: lighten($brand-color, 35%);
+.alert .container {
+    max-width: $page-max-width;
+    margin: 0 auto;
+    padding: 0.75em 1em;
+
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    .message {
+        flex: 1;
+        text-align: center;
+
+        p, h1, h2, h3, h4, h5, h6 {
+            margin: 0;
+            font-size: 1em;
         }
     }
-    &:not(.important) + &:not(.important) .container {
-        margin-top: 2px;
+
+    .dismiss_button {
+        border: 1px solid transparent;
+        border-color: none;
+        background: none;
+        color: inherit;
+        padding: 0.4em;
+        height: 1.5em;
+        width: 1.5em;
+
```

Comment:
cursor: pointer

Reply:
Historically, `cursor: pointer` has only been used for links. It's obvious this isn't a link so using it here wouldn't be a problem, but I don't think it necessarily needs more affordance (people know what an X button does).

Reply:
If anything, I would add a subtle background on hover.